prusti-contracts/prusti-specs/Cargo.toml

```diff
@@ -15,3 +15,4 @@ uuid = { version = "1.0", features = ["v4", "serde"] }
 serde_json = "1.0"
 serde = { version = "1.0", features = ["derive"] }
 prusti-utils = { path = "../prusti-utils" }
+itertools = "0.10.3"
```

Comment:
What is it used for?

Reply:
It is used here: https://github.com/viperproject/prusti-dev/pull/1061/files#diff-1a796772a67353a35d030a434d178f030a5899ced005a35ea104780ee92fd9b3R95. The unique() filters out any duplicate parameters of print_counterexample. Otherwise this would result in a syntax error.